client/package.json

```diff
@@ -24,6 +24,7 @@
     "@angular/platform-browser-dynamic": "~13.0.0",
     "@angular/router": "~13.0.0",
     "rxjs": "~7.4.0",
+    "scriptjs": "^2.5.9",
```

Comment:
I would think twice about introducing a dependency for just that one call. We could use something like

```
	public loadScript(url: string) {
		const body = <HTMLDivElement> document.body;
		cosnt script = document.createElement('script');
		script.innerHTML = '';
		script.src = url;
		script.async = true;
		script.defer = true;
		body.appendChild(script);
	}
```

to achieve the same thing, I think.

Reply:
Done